moped-editor/src/utils/signalComponentHelpers.js

```diff
@@ -214,17 +217,10 @@ export const generateProjectComponent = (
     description: componentDef.description,
     component_id: componentDef.component_id,
     status_id: 1,
-    moped_proj_features_components: {
+    moped_proj_features: {
       data: featureCollection.features.map(feature => ({
-        name: componentDef.component_name,
-        description: componentDef.description,
         status_id: 1,
-        moped_proj_feature_object: {
-          data: {
-            status_id: 1,
-            location: feature,
-          },
-        },
+        feature: feature,
```

Comment:
linting (L116) and otherwise fixing up the graphql types to align with the schema changes.